compiler/bindings/python/test/build_api/mnist_builder_test.py

```diff
@@ -92,11 +99,24 @@ def testActionCLArg(self):
                 args=[
                     "--mnist-onnx-url",
                     "https://github.com/iree-org/doesnotexist",
-                ],
+                ]
+                + DEFAULT_TARGET_ARGS,
                 stdout=out_file,
                 stderr=err_file,
             )
 
+    def testBuildNonDefaultSubTarget(self):
+        mod = load_build_module(THIS_DIR / "mnist_builder.py")
+        out_file = io.StringIO()
+        iree_build_main(
+            mod, args=["mnist/mnist.mlir"] + DEFAULT_TARGET_ARGS, stdout=out_file
+        )
+        output = out_file.getvalue().strip()
+        self.assertIn("genfiles/mnist/mnist.mlir", output)
```

Comment:
This fails on Windows, since `output` is a raw string that includes `\\` characters

Fix:
```diff
diff --git a/compiler/bindings/python/test/build_api/mnist_builder_test.py b/compiler/bindings/python/test/build_api/mnist_builder_test.py
index fb8cefbecf..7b1f641110 100644
--- a/compiler/bindings/python/test/build_api/mnist_builder_test.py
+++ b/compiler/bindings/python/test/build_api/mnist_builder_test.py
@@ -112,8 +112,8 @@ class MnistBuilderTest(unittest.TestCase):
             mod, args=["mnist/mnist.mlir"] + DEFAULT_TARGET_ARGS, stdout=out_file
         )
         output = out_file.getvalue().strip()
-        self.assertIn("genfiles/mnist/mnist.mlir", output)
         output_path = Path(output)
+        self.assertIn("genfiles/mnist/mnist.mlir", output_path.as_posix())
         contents = output_path.read_text()
         self.assertIn("module", contents)
```